apps/board/src/app.module.ts

```diff
@@ -25,7 +27,11 @@ import mysqlConfig from './config/mysql.config';
           logging: true,
           synchronize: process.env.NODE_ENV === 'development',
         };
-        return boardTypeOrmModuleOptions;
+      },
+      dataSourceFactory: async (options: DataSourceOptions): Promise<DataSource> => {
+        if (!options) throw new Error('Invalid options passed');
+        const dataSource = new DataSource(options);
+        return addTransactionalDataSource(dataSource);
       },
     }),
```

Comment:
```suggestion
      dataSourceFactory: async (option?) => {
        if (!option) {
          throw new Error('Option is required');
        }

        return addTransactionalDataSource(new DataSource(option));
      },
```

위와 같은 식으로 간단하게 수정해도 될 것 같습니다!

Transactional 관련 PR 리뷰 내용입니다
[PR: 31 ](https://github.com/GrebbMe/grabbme-be/pull/31/files/a823912e42eb7b02f3d7289ff8211fc1b09b1684)

Reply:
수정된 커밋입니다 : 01dbb08178ca3a023d60a5b7528406f3fed6a72e
감사합니다~!